XamlControlsGallery/ControlPagesSampleCode/XamlUICommand/XamlUICommandSample1_xaml.txt

```diff
@@ -0,0 +1,16 @@
+﻿<Page.Resources>
+    <XamlUICommand x:Name="CustomXamlUICommand" ExecuteRequested="CustomXamlUICommand_ExecuteRequested" Description="This is a custom command" Label="Custom Command">
```

Comment:
Nit: can you wrap this line, maybe starting with Description, to make it easier to read in the gallery?